docs/documentation/securing_apps/topics/oidc/fapi-support.adoc

```diff
@@ -27,9 +27,9 @@ In case you want to use <<_backchannel_authentication_endpoint,CIBA>> in a FAPI
 There is a need to use the `fapi-1-advanced` profile, or other client profile containing the requested executors, as the `fapi-ciba` profile contains just CIBA-specific executors.
 When enforcing the requirements of the FAPI CIBA specification, there is a need for more requirements, such as enforcement of confidential clients or certificate-bound access tokens.
 
-==== Open Banking Brasil Financial-grade API Security Profile
+==== Open Finance Brasil Financial-grade API Security Profile
 
-{project_name} is compliant with the https://openbanking-brasil.github.io/specs-seguranca/open-banking-brasil-dynamic-client-registration-1_ID2-ptbr.html[Open Banking Brasil Financial-grade API Security Profile 1.0 Implementers Draft 2].
+{project_name} is compliant with the https://openbanking-brasil.github.io/specs-seguranca/open-banking-brasil-financial-api-1_ID3.html[Open Finance Brasil Financial-grade API Security Profile 1.0 Implementers Draft 3].
```

Comment:
When I visit the URL, it mentions this page is outdated, and redirects to 
https://openfinancebrasil.atlassian.net/wiki/spaces/OF/pages/82083996/EN+Open+Finance+Brasil+Financial-grade+API+Security+Profile+1.0+Implementers+Draft+3

So should the link be updated?

Reply:
Yes, https://openfinancebrasil.atlassian.net/wiki/spaces/OF/pages/82083996/EN+Open+Finance+Brasil+Financial-grade+API+Security+Profile+1.0+Implementers+Draft+3 is better. I will fix the PR.